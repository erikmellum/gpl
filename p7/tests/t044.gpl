string str = "hello";
int i = 42;

on akey
{
    str += i;
}

on skey
{
    print("str = " + str);
}

