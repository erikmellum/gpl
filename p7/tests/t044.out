gpl.cpp::main()
  input file(tests/t044.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int i 42
string str "hello"
gpl.cpp::main() Calling window->initialize().
gpl.cpp::main() Passing control to window->main_loop().
gpl[11]: str = hello42

Printing the symbol table from signal_handler()
int i 42
string str "hello42"
